test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Metric/MetricTests.cs

```diff
@@ -64,6 +65,18 @@ public async Task GivenHealthCheckPath_WhenInvoked_MetricNotificationsNotEmitted
                 (type: typeof(CosmosStorageRequestMetricsNotification), count: 2, resourceType: (string)null));
         }
 
+        [Trait(Traits.Priority, Priority.One)]
+        [Fact]
+        public async Task GivenABatch_WhenInvoked_MetricNotificationsShouldBeEmitted()
+        {
+            _metricHandler?.ResetCount();
+
+            await ExecuteAndValidate(
+                () => _client.CreateAsync(Samples.GetDefaultBatch().ToPoco()),
```

Comment:
I think this is going to just create a bundle on the server and not actually do a batch. You most likely need to do `_client.PostBundleAsync`